core/src/com/unciv/logic/city/CityReligion.kt

```diff
@@ -57,7 +119,11 @@ class CityInfoReligionManager: Counter<String>() {
         for (city in allCitiesWithin10Tiles) {
             val majorityReligionOfCity = city.religion.getMajorityReligion()
             if (majorityReligionOfCity == null) continue
-            else add(majorityReligionOfCity, 6) // todo - when holy cities are implemented, *5
+            else addPressure(
+                majorityReligionOfCity, 
+                if (city.isHolyCity()) 11
```

Comment:
So the comment was wrong about 6 **`*`** 5, as this is `+` - OK I don't know the sources